apps/server/deploy.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+cp ../../yarn.lock .
+yarn build
+git init
+heroku git:remote -a bugzilla-dashboard-temp-server
+git add build package.json yarn.lock phonebook.json Procfile
+git commit -am "Deploy server to Heroku"
+git push heroku master -f
```

Comment:
I think I tinkered with this idea before I split off firefox-health into two different repos.
Does this require a human to have to run the deploy command?